evdns.c

```diff
@@ -3562,6 +4222,14 @@ evdns_base_set_option_impl(struct evdns_base *base,
 		if (!(flags & DNS_OPTION_MISC)) return 0;
 		log(EVDNS_LOG_DEBUG, "Setting SO_SNDBUF to %s", val);
 		base->so_sndbuf = buf;
+	} else if (!strcmp(option, "use-vc")) {
+		if (!(flags & DNS_OPTION_MISC)) return 0;
+		log(EVDNS_LOG_DEBUG, "Setting use-vc option");
+		base->global_tcp_flags |= DNS_QUERY_USEVC;
+	} else if (!strcmp(option, "ingore-tc")) {
```

Comment:
typo here, can you add a smoke test for this namings?

Reply:
Done